components/match2/wikis/brawlstars/match_group_input_custom.lua

```diff
@@ -345,15 +329,7 @@ end
 
 function mapFunctions.getTournamentVars(map)
 	map.mode = Logic.emptyOr(map.mode, Variables.varDefault('tournament_mode', '3v3'))
```

Comment:
Note: The default mode mismatches in this game between map & match (unrelated to PR, just something to fix sometime)

Reply:
asked in #brawlstars what they want:
https://discord.com/channels/93055209017729024/591393837927366672/1006093521847074888

Reply:
they want `'team'`